Dockerfile

```diff
@@ -40,6 +40,9 @@ RUN dnf install -y python3 python3-requests && \
     dnf clean all && \
     rm -rf /var/cache/{yum,dnf}/*
 
+# install vendor driver libs
+RUN dnf install python3-ibmcclient
+
```

Comment:
please include this as part of the previous RUN command. It is good practice to limit the number of RUN and in addition you would benefit from the cache cleanup.